packages/blocks/src/api/parser.js

```diff
@@ -25,6 +25,26 @@ import { getCommentDelimitedContent } from './serializer';
 import { attr, html, text, query, node, children, prop } from './matchers';
 import { normalizeBlockType } from './utils';
 
+/**
+ * Options for parse.
+ *
+ * @property {Object<string,Function>} sources An object defining custom source
```

Comment:
Hi @aduth,
>  the value a function receiving an attribute schema and expected to return the desired value.

Would it make sense for the function to also receive the innerHTML plus the commentAttributes?

Imagining the following scenarios:
	- an attribute that is derived from the text in a given HTML element.
	- A product block that contains the id of the product as a comment, and we want to query some attributes of that product using an external REST API.